database/seeds/DatabaseSeeder.php

```diff
@@ -13,4 +13,4 @@ public function run()
     {
        $this->call(CategoryTableSeeder::class);
     }
-}
+}
```

Comment:
Ở đây e không call toàn bộ seeder vào, nên chị pull về ko chạy seed được. Chỗ này phải fix e nha :)